python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -266,10 +266,9 @@ def predict(self, X, output_class=True, algo='auto', threshold=0.5,
         ----------
         y : Dask cuDF dataframe or CuPy backed Dask Array (n_rows, 1)
         """
-        if self.num_classes > 2 or predict_model == "CPU":
-            preds = self.predict_model_on_cpu(X,
-                                              convert_dtype=convert_dtype)
-
+        if predict_model == "CPU":
+            preds = self._predict_using_cpu(X,
```

Comment:
The cpu predict should call the function : `predict_model_on_cpu` and not `_predict_using_cpu`. `_predict_using_cpu` is the static function called inside `predict_model_on_cpu`